packages/components/tests/dummy/app/templates/components/modal.hbs

```diff
@@ -0,0 +1,760 @@
+{{page-title "Modal Component"}}
+
+<h2 class="dummy-h2">Modal</h2>
+
+<section>
+  <h3 class="dummy-h3" id="overview"><a href="#overview" class="dummy-link-section">§</a> Overview</h3>
+
+  <p class="dummy-paragraph">Modals are used to request information or feedback from the user, confirm a decision, or
+    less frequently to provide additional context about a function or feature.</p>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="component-api"><a href="#component-api" class="dummy-link-section">§</a> Component API</h3>
+  <p class="dummy-paragraph" id="component-api-modal">
+    Here is the API for the component:
+  </p>
+  <dl class="dummy-component-props" aria-labelledby="component-api-modal">
+    <dt>
+      size
+      <code>enum</code>
+    </dt>
+    <dd>
+      <p>
+        Sets the width of the modal.
+      </p>
+
+      <p>Acceptable values:</p>
+      <ol>
+        <li>small</li>
+        <li class="default">medium</li>
+        <li>large</li>
+      </ol>
+    </dd>
+
+    <dt>color <code>enum</code></dt>
+    <dd>
+      <p>
+        Sets the color scheme for the modal header elements: icon, tagline and title.
+      </p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">neutral</li>
+        <li>warning</li>
+        <li>critical</li>
+      </ol>
+    </dd>
+    <dt>onOpen <code>function</code></dt>
+    <dd>
+      <p>
+        Callback function invoked when the modal is opened.
+      </p>
+    </dd>
+    <dt>onClose <code>function</code></dt>
+    <dd>
+      <p>
+        Callback function invoked when the modal is closed.
+      </p>
+    </dd>
+    <dt>
+      isDismissDisabled
+      <code>boolean</code>
+    </dt>
+    <dd>
+      <p>Default: <span class="default">false</span></p>
+      <p>Set this boolean to
+        <code class="dummy-code">true</code>
+        if you want to prevent the modal from being closed (for instance, to avoid accidental data loss in an
+        unsubmitted form). Make sure you communicate to users the reason why the modal is still open.
```

Comment:
Maybe add something like this? (it will need to be formatted or the linter will complain)

```suggestion
        unsubmitted form). Make sure you communicate to users the reason why the modal is still open, and what they need to do to resolve the problem that is preventing the modal to be closed.
```

Reply:
(feel free to reword it :) )

Reply:
Thank you!